package.json

```diff
@@ -67,6 +67,7 @@
     "ft_management": "cd tests/functional/report && yarn test",
     "ft_node": "cd tests/functional/raw-node && yarn test",
     "ft_node_routes": "cd tests/functional/raw-node && yarn run test-routes",
+    "ft_node_route_backbeat": "cd tests/functional/raw-node && mocha --reporter mocha-multi-reporters --reporter-options configFile=$INIT_CWD/tests/reporter-config.json -t 40000 test/routes/routeBackbeat.js",
```

Comment:
Do we need to keep this specific test ? 
Otherwise LGTM

Reply:
Sorry I should have mentioned the commits that are meant to be dropped, I won't be adding these to the `package.json`